Tasks/Ivanow/Task7/Task7_back.py

```diff
@@ -0,0 +1,77 @@
+
+class Movie:
+    movies_count = 0
+
+    def __init__(self):
+        Movie.movies_count += 1
+        self.movies_count = Movie.movies_count
+        self.title = input('Input title: ')
+        self.director = input('Input director: ')
+        self.year = input('Input year: ')
+        self.genre = input('Input genre: ')
+
+    def displayMovie(self):
+        print(f'Title: {self.title}\tDirector: {self.director}\tYear:{self.year}\tGenre:{self.genre}')
```

Comment:
communication with a user should be a front module responsibility, it's wrong to put this call here in the back